src/System.Diagnostics.TraceSource/tests/DefaultTraceListenerClassTests.cs

```diff
@@ -27,6 +28,11 @@ public override void Write(string message)
             {
                 _writer.Write(message);
             }
+
+            public override void WriteLine(string message)
+            {
+                _writer.WriteLine(message);
+            }
```

Comment:
The public method `Write` in desktop, just calls the private `Write` API. We deleted that step in Core and just do what the private API did inside the public one. Was I clear?